buildroot/package/audiocontrol2/audiocontrol2.conf

```diff
@@ -35,6 +35,19 @@ url=http://127.0.0.1:80/sound/volume
 external_metadata=1
 
 [controller:ac2.plugins.control.keyboard.Keyboard]
+# keycodes for OSMC remote - https://osmc.tv/store/product/osmc-remote-control/
+12 = volume_down
+13 = volume_up
+23 = mute
+28 = playpause
+103 = previous
+105 = previous
+106 = next
+108 = next
+114 = volume_up
+115 = volume_down
+128 = stop
+164 = playpause
```

Comment:
Please do not overwrite the default configuration.

Reply:
Sorted